File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_UserGroups.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_UserGroups } from './Sys_UserGroups.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_UserGroupsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'User_Id', 'Group_Id'];

    List: Sys_UserGroups[] | undefined;
    ListFilter: Sys_UserGroups[] | undefined;
    FormData!: Sys_UserGroups;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_UserGroups";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_UserGroupsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_UserGroupsService : BaseService<Sys_UserGroups, ISys_UserGroupsRepository>
    , ISys_UserGroupsService
    {
    private readonly ISys_UserGroupsRepository _Sys_UserGroupsRepository;
    public Sys_UserGroupsService(ISys_UserGroupsRepository Sys_UserGroupsRepository) : base(Sys_UserGroupsRepository)
    {
    _Sys_UserGroupsRepository = Sys_UserGroupsRepository;
    }
    }
    }

